doc/source/whatsnew/v1.0.2.rst

```diff
@@ -23,6 +23,7 @@ Fixed regressions
 - Fixed regression where :func:`read_pickle` raised a ``UnicodeDecodeError`` when reading a py27 pickle with :class:`MultiIndex` column (:issue:`31988`).
 - Fixed regression in :class:`DataFrame` arithmetic operations with mis-matched columns (:issue:`31623`)
 - Fixed regression in :meth:`GroupBy.agg` calling a user-provided function an extra time on an empty input (:issue:`31760`)
+- Fixed regression in :meth:`loc.__setitem__` throwing an error when a datetime64[ns, tz] value is provided (:issue:`32395`)
```

Comment:
loc isn't a toplevel method. 

```suggestion
- Fixed regression in :meth:`DataFrame.loc` throwing an error when a ``datetime64[ns, tz]`` value is provided (:issue:`32395`)
```

Reply:
The change is at the block level, it affects both `DataFrame.loc` and `Series.loc`, I'll update the file.